TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Owner: {Your Name} {you@example.com}, {@you} on the Elm Slack
```

Comment:
"Author" doesn't immediately convey what I'm getting at. It's more like responsible party / mini-bdfl. Coordinator? Or if we wanna get fancy, coördinator?